### ISSUE TITLE
Flaky spec: Poll Officing Officing dashboard available for multiple sessions

### ISSUE THREAD

**Issue by bertocq** [AyuntamientoMadrid/consul#1200]
Warning!
===
This spec probably only fails between 00:00 and 02:00 am Madrid time

# What
Tests that fail randomly are called "flakies", this one seems to be one:

**Randomized seed:** 24781

**Failure:**

```
  1) Poll Officing Officing dashboard available for multiple sessions
     Failure/Error: select 'DNI', from: 'residence_document_type'
     Capybara::ElementNotFound:
       Unable to find visible select box "residence_document_type" that is not disabled
     # ./spec/features/officing_spec.rb:160:in `block (3 levels) in <top (required)>'
     # ./spec/sessions_helper.rb:5:in `in_browser'
     # ./spec/features/officing_spec.rb:156:in `block (2 levels) in <top (required)>'
```

# How
- [ ] Explain why the test is flaky, or under which conditions/scenario it fails randomly
- [ ] Explain why your PR fixes it
- [ ] Create a backport PR to consul/consul when the fixing PR is approved

### Random values issues
If the problem comes from randomly generated values, running multiple times a single spec could help you reproduce the failure by running at your command line:
```bash
for run in {1..10}
do
  bin/rspec ./spec/features/budgets/investments_spec.rb:256
done
```

You can also try running a single spec in Travis:
Add option `:focus` to the spec and push your branch to Github, for example:
```ruby
scenario 'Show', :focus do
```

But remember to remove that `:focus` changes afterwards when submitting your PR changes!

### Test order issues
Running specs in the order they failed may discover that the problem is that a previous test sets an state in the test environment that makes our flaky fail/pass. Tests should be independent from the rest.

After executing rspec you can see the seed used, add it as an option to rspec, for example:
`bin/rspec --seed 55638` (check **Randomized seed** value at beginning of issue)

### Other things to watch for
- Time related issues (current time, two time or date comparisons with miliseconds/time when its not needed)
- https://semaphoreci.com/community/tutorials/how-to-deal-with-and-eliminate-flaky-tests

**Comment by bertocq** [comment]
Happened again at https://travis-ci.org/AyuntamientoMadrid/consul/jobs/338257531

**Comment by bertocq** [comment]
happened at https://travis-ci.org/AyuntamientoMadrid/consul/jobs/338260739

**Comment by raul-fuentes** [comment]
I'll do this one

**Comment by bertocq** [comment]
Great work @raul-fuentes! We'll need a final backport to consul to be able to close this issues 👍

**Comment by raul-fuentes** [comment]
I'll look into it

**Comment by javierm** [comment]
It's failed recently, ran at ~01:00: [Travis build 10242, job 9](https://travis-ci.org/AyuntamientoMadrid/consul/jobs/422316154).

**Comment by javierm** [comment]
Closed via consu#2712 and #1625.
